dapps/react-dapp-v2/src/contexts/JsonRpcContext.tsx

```diff
@@ -642,18 +641,38 @@ export function JsonRpcContextProvider({
       }
     ),
   };
-  // -------- POLKADOT RPC METHODS --------
 
+  // -------- POLKADOT RPC METHODS --------
   const polkadotRpc = {
     testSignTransaction: _createJsonRpcRequestHandler(
       async (
         chainId: string,
         address: string
       ): Promise<IFormattedRpcResponse> => {
-        // Below example is a scale encoded payload for system.remark("this is a test wallet-connect remark") transaction.
-        // decode url: https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Frpc.polkadot.io#/extrinsics/decode/0x00019074686973206973206120746573742077616c6c65742d636f6e6e6563742072656d61726b
-        const transactionPayload =
-          "0x00019074686973206973206120746573742077616c6c65742d636f6e6e6563742072656d61726b05010000222400000d00000091b171bb158e2d3848fa23a9f1c25182fb8e20313b2c1eb49219da7a70ce90c3dc1f37ce7899cf20f63f5ea343f33e9e7b229c7e245049c2a7afc236861fc8b4";
+        const transactionPayload = {
```

Comment:
Nice 🙌  thanks for adding a more human-friendly payload 😄

Reply:
I did not want to add a dependency to Polkadot RPCs for this validation. Polkadot.js apis currently uses the RPC for validating a tx + signature but there might be still some way to just validate the signature which I need to do some research to find out. I'll send a separate PR when I find the offline signature validation.

Reply:
Thanks. Hopefully, this makes it easier to read the transactions, and also helps with testing 🙌